src/deploy/2025-08-07_account_block_accounts.sql

```diff
@@ -0,0 +1,24 @@
+BEGIN;
+
+CREATE FUNCTION vibetype.account_block_accounts()
+RETURNS TABLE (
+  id          UUID,
+  username    TEXT,
+  storage_key TEXT
+) AS $$
+BEGIN
+  RETURN QUERY
+    SELECT a.id, a.username, u.storage_key
+    FROM vibetype.account a
+      JOIN vibetype.account_block b ON a.id = b.blocked_account_id
+      LEFT JOIN vibetype.profile_picture p ON a.id = p.account_id
+      LEFT JOIN vibetype.upload u ON p.upload_id = u.id
+    WHERE b.created_by = vibetype.invoker_account_id();
+END
+$$ LANGUAGE PLPGSQL STRICT STABLE SECURITY DEFINER;
```

Comment:
I think we can use a simple `SQL` language here which is more transparent to the query engine.

```suggestion
    SELECT a.id, a.username, u.storage_key
    FROM vibetype.account a
      JOIN vibetype.account_block b ON a.id = b.blocked_account_id
      LEFT JOIN vibetype.profile_picture p ON a.id = p.account_id
      LEFT JOIN vibetype.upload u ON p.upload_id = u.id
    WHERE b.created_by = vibetype.invoker_account_id();
END
$$ LANGUAGE SQL STRICT STABLE SECURITY DEFINER;
```